extensions/amp-analytics/0.1/requests.js

```diff
@@ -133,10 +139,34 @@ export class RequestHandler {
 
     if (!this.baseUrlPromise_) {
       expansionOption.freezeVar('extraUrlParams');
```

Comment:
Discussed offline, we will not support `${extraUrlParams}` in `requestOrigin` because we will be using https://developer.mozilla.org/en-US/docs/Web/API/URL/URL to construct the URL, which removes extra information after the hostname.